app-connector-client/server/config.js

```diff
@@ -56,6 +56,9 @@ function configValidation(configJson) {
             }
         }
     }
+    if (configJson.logo && !configJson.logo.match(/^.+\.(svg)$/)) {
+        error_message += "\nlogo image must be in svg format"
```

Comment:
Repeat the actual value, that helps finding the problem. Print the actual pattern to easily see where the problem is

```suggestion
        error_message += "\nlogo " + configJson.logo + " must be a path to an image in svg format, matching pattern '^.+\.(svg)$'"
```